### DISCUSSION TITLE
any real examples of next.js Link tag?

### DISCUSSION THREAD

**Discussion by guidorice** [markdoc/markdoc#305] [Q&A] [answered]
I've got the `link` tag all wired up in my app via this code:

```typescript
export { comment, head, link, script } from '@markdoc/next.js/tags';
```

The link tag renders OK (sans styling), however I get a react hydration error in my dev environment:
```
{% link href="/mypage" %}
    test next.js link
{% /link %}
```

> Error: Hydration failed because the initial UI does not match what was rendered on the server.

Environment:

* `npm run dev`
* next 13.0.1
* @markdoc/next.js@0.2.0
* @markdoc/markdoc@0.1.13

I tried reproducing this in the markdoc-starter repo, but have not come up with a good bug reproduce case yet. My project is based on the markdoc starter repo. Anyone else seen this? Any open source code showing how you've used the next.js link tag in markdoc?

**Comment by mfix-stripe** [comment, marked as answer]
@guidorice this change fixed this issue for me with the latest version of the starter:
```js
import Link from 'next/link';
import {link as nextLink} from '@markdoc/next.js/tags';

export * from './callout.markdoc';
export {comment, head, script} from '@markdoc/next.js/tags';

export const link = {
  ...nextLink,
  render: Link,
};
```
